app/views/groups/_add_mentor_modal.html.erb

```diff
@@ -29,7 +29,7 @@
             <%= form.select :emails, [], {}, { class: "form-select", id: "group_mentor_emails", multiple: true } %>
           </div>
           <div class="modal-footer">
-            <%= form.submit "Add mentors", class: "btn primary-button", id: "add-mentor-button" %>
+            <%= form.submit "Add mentors", class: "btn primary-button add-mentor-button" %>
```

Comment:
Instead of making `add-mentor-button` and `add-member-button` as class name, keep it as ID because it signifies a single element only